lib/src/model/package_graph.dart

```diff
@@ -829,22 +822,23 @@ class PackageGraph {
   /// This is used when we might need a Library object that isn't actually
   /// a documentation entry point (for elements that have no Library within the
   /// set of canonical Libraries).
-  Library findOrCreateLibraryFor(ResolvedLibraryResult result) {
+  Library findOrCreateLibraryFor(DartDocResolvedLibrary resolvedLibrary) {
+    final elementLibrary = resolvedLibrary.result.element.library;
```

Comment:
might consider having some helper getters in DartDocResolvedLibrary since we control that interface, now.

Reply:
Good suggestion! Done.